Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -23,6 +23,9 @@ struct PlaceUtilities {
 
     static func filterPlacesForCarousel(_ places: [Place]) -> [Place] {
         return places.filter { place in
+            // always show places if they have events
+            guard place.events.isEmpty else { return true }
```

Comment:
Yes, only events that are to be shown will appear on places. If an event shouldn't be shown, it will no appear on a place as they are filtered out in the `EventProvider`